Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/AssetGroups/AssetAssetGroup.tsx

```diff
@@ -27,11 +27,13 @@ import * as _ from 'lodash';
 import { useHistory } from 'react-router-dom';
 import Table from '@gpa-gemstone/react-table';
 import { AssetTypeSlice } from '../Store/Store';
+import { OpenXDA } from '@gpa-gemstone/application-typings';
```

Comment:
is this needed? There seems to be no changes requiring this

Reply:
Removed.